content/help/ops/telemetry/envoy-stats/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Envoy Statistics
+description: Fine-grained control of Envoy statistics.
+weight: 95
+---
+
+The Envoy proxy keeps detailed statistics about network traffic.
+
+Envoy's statistics only cover the traffic for a particular Envoy instance.  See
+[Telemetry](/docs/tasks/telemetry/) for persistent per-service Istio telemetry.  The
+statistics recorded by Envoy can provide more information about specific pod instances.
+
+To see the statistics for a pod:
+
+{{< text bash >}}
+$ kubectl exec -it $POD  -c istio-proxy  -- sh -c 'curl localhost:15000/stats'
+{{< /text >}}
+
+See [the Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/cluster_manager/cluster_stats) for an explanation of the data recorded.
+
+By default, Istio configures Envoy to record minimal statistics.  The default collection
+keys are `cluster_manager`, `listener_manager`, `http_mixer_filter`, `tcp_mixer_filter`, `server`, and `cluster.xds-grpc`.  To see the Envoy settings for statistics data collection use
+`istioctl proxy-config bootstrap` following [these instructions](https://istio.io/help/ops/traffic-management/proxy-cmd/).
+Consult the `statsMatcher` JSON element.
+
+Configure Envoy to record statistics for inbound or outbound traffic by adding the
+`sidecar.istio.io/statsInclusionPrefixes` annotation to the pod template in a Kubernetes `Deployment`.
+Add `cluster.outbound` for gather data about outbound traffic activity and circuit breaking.
+To gather data on inbound traffic, add `listener`.  A sample annotation including `cluster.outbound`
+can be seen in `samples/httpbin/sample-client/fortio-deploy.yaml`.
```

Comment:
```suggestion

{{< text_file file="/samples/httpbin/sample-client/fortio-deploy.yaml" syntax="yaml">}}
```

Reply:
This suggestion gives me ```
Error: Error building site: "/site/content/help/ops/telemetry/envoy-stats/index.md:42:1": failed to render shortcode "text_file": failed to process shortcode: "/site/layouts/shortcodes/text_file.html:7:13": execute of template failed: template: shortcodes/text_file.html:7:13: executing "shortcodes/text_file.html" at <readFile (.Get "file...>: error calling readFile: stat /site/content_zh/samples/httpbin/sample-client/fortio-deploy.yaml: no such file or directory
```  I will try `<github_file>` instead.

Reply:
```suggestion
can be seen in [fortio-deploy.yaml]({{< github_file >}}/samples/httpbin/sample-client/fortio-deploy.yaml`).
```